server/src/index.ts

```diff
@@ -47,9 +51,35 @@ app.use((err: any, req: Request, res: Response, next: NextFunction) => {
   res.render('error');
 });
 
-const port = process.env.PORT || 5000; // Use environment variable or default to 3001
-app.listen(port, () => {
-  console.log(`Server running on http://localhost:${port}`);
+// const port = process.env.PORT || 3001; // Use environment variable or default to 3001
+// app.listen(port, () => {
+//   console.log(`Server running on http://localhost:${port}`);
+// });
+
+// Logging the DB connection and starting server
+const startServer = async () => {
+  try {
+    // Connect to the database
+    await prisma.$connect(); 
+    console.log('The logger: Connected to the database successfully!'); 
+
+    // Start the server
+    const port = process.env.PORT || 3001;
+    app.listen(port, () => {
+      console.log(`Server running on http://localhost:${port}`);
+    });
+  } catch (error) {
```

Comment:
In your catch block, you are logging generic error messages.
Possible solution: We can add more context to the error logs, such as timestamps or details about which part of the code failed.